src/common/stats/StatsManager.h

```diff
@@ -65,7 +65,9 @@ class StatsManager final {
     static void setReportInfo(HostAddr addr, int32_t interval);
 
     // Both register methods return the index to the internal data structure.
-    // This index will be used by addValue() methods
```

Comment:
Why change this?The metrics is read/write concurrency by server handlers.

Reply:
The `register` method is not for read/write, only for server to reigster stats.